olympics/olympics.ipynb

```diff
@@ -0,0 +1,240 @@
+   "source": [
+    "df = df.dropna(subset=[\"Weight\", \"Height\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a802c22",
+   "metadata": {},
+   "source": [
+    "One of the first things to look at is the the average height of each sport. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5786c4f5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "avg_height_by_sport = df.groupby('Sport')['Height'].mean()\n",
+    "avg_height_by_sport.sort_values(inplace=True)\n",
+    "avg_height_by_sport.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93367f84",
+   "metadata": {},
+   "source": [
+    "Using hvplot, we can create a horizontal bar chart by using `.hvplot.barh`. Then we add labels to our chart and select the basketball and gymnastics as they have the highest and lowest heights. We finally combine everything by overlaying the chart, labels, and select together to create our chart through the use of `*` operator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "82f985b4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bar = avg_height_by_sport.hvplot.barh(by='Sport', stacked=True, height=600)\n",
+    "txt = avg_height_by_sport.hvplot.labels(\"Sport\", \"Height\", \"{Height:.0f} cm\", invert=True, text_align=\"left\", text_baseline=\"middle\", text_color=\"darkred\")\n",
+    "bar_sel = bar.select(Sport=[\"Basketball\", \"Gymnastics\"]).opts(fill_color=\"darkred\")\n",
+    "(bar * bar_sel * txt.select(Sport=[\"Basketball\", \"Gymnastics\"])).opts(title=\"Average Height of each Olympic Sport, a 28 cm difference\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3412b2d0",
+   "metadata": {},
+   "source": [
+    "We can also create a line plot between differnece the average heights of basketball and gymanstic olympians for each year. It's interesting to see that the difference has been growing wider with it peaking at 1992. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97e655ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "avg_basketball_height = df[df[\"Sport\"] == \"Basketball\"].groupby(\"Year\")[\"Height\"].mean()\n",
+    "avg_gymnastics_height = df[df[\"Sport\"] == \"Gymnastics\"].groupby(\"Year\")[\"Height\"].mean()\n",
+    "avg_height_diff = avg_basketball_height - avg_gymnastics_height\n",
+    "avg_height_diff.hvplot.line()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a5ab6dd1",
+   "metadata": {},
+   "source": [
+    "We can also take a look at the height and weights at the same time for each medal and sport combination. We were able to overlay the plot earlier using `*` but we can also arrange the two plots together using `+`. To make them vertical we use `.cols(1)`.\n",
+    "\n",
+    "Note that the medal defaults to \"nan\" which won't display any data on the graphs, so it needs to be changed to something else. "
```

Comment:
Instead of this, you can probably get rid of NaNs ahead of time: `df.dropna(subset=["Medal"])`